Gemfile

```diff
@@ -20,7 +20,7 @@ gem "rails", "4.2.1"
 gem "recipient_interceptor"
 gem "refills"
 gem "sass-rails", "~> 5.0"
-gem "simple_form"
+# gem "simple_form"
```

Comment:
If these gems are not necessary, can we just remove them instead of commenting them?